resources/src/main/java/org/robolectric/res/android/ResTable_config.java

```diff
@@ -284,16 +285,16 @@
   public static final int MINORVERSION_ANY = 0;
 
   /** The below constants are from android.content.res.Configuration. */
-  private static final int DENSITY_DPI_UNDEFINED = 0;
+  public static final int DENSITY_DPI_UNDEFINED = 0;
   private static final int DENSITY_DPI_LDPI = 120;
-  private static final int DENSITY_DPI_MDPI = 160;
+  public static final int DENSITY_DPI_MDPI = 160;
   private static final int DENSITY_DPI_TVDPI = 213;
```

Comment:
It's visible only for `DeviceConfig`, and only because it's using `DENSITY_DPI_ANY` and `DENSITY_DPI_NONE`, which aren't included in `DisplayMetrics`, or I'd pull it from there instead.